tests/migration/register_system_in_textmode.pm

```diff
@@ -0,0 +1,42 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register the original system before migration
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use Utils::Architectures;
+use utils;
+use migration;
+use registration;
+use y2_base;
+
+sub run {
+    select_console 'root-console';
+
+    my ($self) = @_;
+
+    # Save VIDEOMODE and SCC_REGISTER vars
+    my $orig_videomode = get_var('VIDEOMODE', '');
+    my $orig_scc_register = get_var('SCC_REGISTER', '');
+
+    # Set vars to make yast_scc_registration work in ncures
+    set_var("VIDEOMODE", 'text');
+    set_var("SCC_REGISTER", 'console');
+    # remember we perform registration on pre-created HDD images
+    set_var('HDD_SP2ORLATER', 1);
+
+    register_system_in_textmode;
+
+    # Restore the old value of VIDEOMODE and SCC_REGISTER
+    # Otherwise the automation will chose textmode when doing offline upgrade
+    set_var("VIDEOMODE", $orig_videomode);
+    set_var("SCC_REGISTER", $orig_scc_register);
+}
```

Comment:
We don't need to do this trick to set it and set if back, just need to remove setting:
ADDON_REGBYCMD
and include in this file only:
```
set_var('HDD_SP2ORLATER', 1);
register_product();
register_addons_cmd();
set_var('IN_PATCH_SLE', 0); (I'm not sure about this one, maybe yes)
```
makes sense?

Reply:
yes